webpack.config.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2007-2015, GoodData(R) Corporation. All rights reserved.
```

Comment:
can't u just merge this into grunt's config definition or, better get the grunt's config and put the stuff in here...?